build.env

```diff
@@ -12,8 +12,8 @@
 CSI_IMAGE_VERSION=canary
 
 # Ceph version to use
-BASE_IMAGE=quay.io/ceph/ceph:v16
-CEPH_VERSION=pacific
+BASE_IMAGE=ceph/ceph:v17
```

Comment:
AFAIK This will not work for the github actions and for local building also as it will try to pull from the docker.io by default.

Reply:
Similar to #2953, but include `quay.io` in the 'short' version of the tag. Pulling the image from the CI mirror can then be done with `docker://registry-ceph-csi.apps.ocp.ci.centos.org/quay.io/ceph/ceph:v17`. I have now manually pushed `:v17` and `:v16` to the CI registry so that testing is a little quicker (mirroring is done nightly).

Reply:
This seems to work now, great!